src/Surfnet/StepupRa/RaBundle/Controller/RaManagementController.php

```diff
@@ -50,7 +49,7 @@ public function manageAction(Request $request)
         $institution = $this->getUser()->institution;
         $logger->notice(sprintf('Loading overview of RA(A)s for institution "%s"', $institution));
 
-        $searchQuery = (new RaListingSearchQuery($institution, 1))
+        $searchQuery = (new RaListingSearchQuery($this->getUser()->id, $institution, 1))
```

Comment:
Do you have a clue why the page number is hard coded to 1? It seems to potentially limit the results this action is able to return if a large amout of RA's become available (which is a more realistic possibility). I'm aware this was already configured like this. Possible bug?

Reply:
@MKodde This could potentially be a bug but it could also be a feature. Because for ra candidates it's implemented differently and it looks like it should work there as expected. Also for the listing no pagination is rendered but for candidates it is? I'll create a issue to clearify this.

Reply:
https://www.pivotaltracker.com/story/show/163859942